common/transport/http/src/rest_api_client.ts

```diff
@@ -109,7 +113,60 @@
     let httpHeaders: any = headers || {};
     if (this._config.sharedAccessSignature) {
       httpHeaders.Authorization = (typeof(this._config.sharedAccessSignature) === 'string') ? this._config.sharedAccessSignature as string : (this._config.sharedAccessSignature as SharedAccessSignature).extend(anHourFromNow());
+      this.executeBody(requestBody, httpHeaders, headers, method, path, timeout, requestOptions, done);
+    } else if (this._config.tokenCredential) {
+      let accessToken = this.getToken();
+      Promise.resolve(accessToken).then((value) => {
```

Comment:
What happens if this rejects?

Reply:
I'm thinking if this rejects, the program will crash.  If not now, then under node 14?

Reply:
Error handling added